hikari/impl/shard.py

```diff
@@ -325,110 +436,28 @@ def intents(self) -> typing.Optional[intents_.Intents]:
 
     @property
     def is_alive(self) -> bool:
-        return self._connected_at is not None
-
-    @property
-    def proxy_settings(self) -> config.ProxySettings:
-        return self._proxy_settings
-
-    @property
-    def sequence(self) -> typing.Optional[int]:
-        return self._seq
-
-    @property
-    def session_id(self) -> typing.Optional[str]:
-        return self._session_id
-
-    @property
-    def session_uptime(self) -> float:
-        return date.monotonic() - self._session_started_at if self._session_started_at is not None else 0
+        return self._run_task is not None and not self._run_task.done()
 
     @property
     def shard_count(self) -> int:
         return self._shard_count
 
-    @property
-    def version(self) -> int:
-        return self._version
+    async def close(self) -> None:
```

Comment:
Should we not close the ratelimiters here plus chunker and other stuff or am i missing something?

Reply:
Fixed.